docs/notifications/schema.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
I think this is missing a few properties before it could be used to actually validate the feed json, but I don't think it'll take much to get there. See the [region json schema file](https://github.com/cds-snc/covid-alert-app/blob/eb5214a3ed60fe47990bef41d89088029e1bbfab/src/locale/translations/regionSchema.json) for example.